airflow/models/xcom.py

```diff
@@ -63,7 +63,7 @@ def init_on_load(self):
         i.e automatically deserialize Xcom value when loading from DB.
         """
         try:
-            self.value = XCom.orm_deserialize_value(self)
+            self.value = self.orm_deserialize_value(self)
```

Comment:
Can this be `self.orm_deserialize_value()`?

Reply:
Yeah, make sense, I didn't get it earlier. Done in 5ad38dc